CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Changelog
 
+All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/docker-mailserver/docker-mailserver/compare/v11.2.0...HEAD)
+
+> **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
+
+## [11.2.0](https://github.com/docker-mailserver/docker-mailserver/releases/tag/v11.2.0)
+
+### Summary
+
+This release features a lot of small and medium-sized changes, many related to how the image is build and tested during CI. The build now requires Docker Buildkit as the ClamAV Signatures are added via `COPY --link ...` during build-time. Moreover, the build is now multi-stage. `ENABLE_LDAP` is now deprecated.
+
+### Added
+
+- **documentation**: improve cron tasks documentation and fix link in documentation
+- **documentation**: added link to brakkee.org for setup of docker-mailserver on Kubernetes
+- **CI**: better build caching for CI
```

Comment:
```suggestion
- **CI**: better build caching for CI ([#2742](https://github.com/docker-mailserver/docker-mailserver/pull/2742))
```

There are no more links to the corresponding PR, was that on purpose? When I read any change logs, sometimes I like to check, what exactly have changed. Having a direct link to the PR makes it easy.

Reply:
> When I read any change logs, sometimes I like to check, what exactly have changed. Having a direct link to the PR makes it easy.

:100: